doc/man7/EVP_PKEY-EC.pod

```diff
@@ -0,0 +1,64 @@
+=pod
+
+=head1 NAME
+
+EVP_PKEY-EC - EVP_PKEY EC keytype and algorithm support
+
+=head1 DESCRIPTION
+
+The B<EC> keytypes is implemented in OpenSSL's default provider.
```

Comment:
NIT: keytype

Reply:
Doh!

Reply:
I would not expect this to restart the approval.

Reply:
Done.

I imagine myself picking those with a fly swat :smile: [SMACK]